web/src/HUD.tsx

```diff
@@ -56,6 +56,10 @@ type HudProps = {
   interfaceVersion: InterfaceVersion
 }
 
+// Snapshot logs are capped to 2MB (max upload size is 4MB; this ensures between the rest of state and JSON overhead
+// that the snapshot should still fit)
+const maxSnapshotLogSize = 2 * 1000 * 1000;
```

Comment:
can we make it more like 1MB? metadata overhead can be pretty high!

Reply:
Yep, lowered in 0c708297!